tool/tsh/common/mcp_db.go

```diff
@@ -226,9 +234,217 @@ func (c *mcpDBStartCommand) prepareDatabases(
+	config := claude.NewConfig()
+	// Since the database is being added to a "fresh" config file the database
+	// will always be new and we can ignore the additional message as well.
+	if _, _, err := m.addDatabaseToConfig(config, dbURI); err != nil {
+		return trace.Wrap(err)
+	}
+
+	w := m.cf.Stdout()
+	if _, err := fmt.Fprintln(w, "Here is a sample JSON configuration for launching Teleport MCP servers:"); err != nil {
+		return trace.Wrap(err)
+	}
+	if err := config.Write(w, claude.FormatJSONOption(m.clientConfig.jsonFormat)); err != nil {
+		return trace.Wrap(err)
+	}
+	if _, err := fmt.Fprintln(w, ""); err != nil {
+		return trace.Wrap(err)
+	}
+	return trace.Wrap(m.clientConfig.printHint(w))
+}
+
+// TODO(gabrielcorado): support updating multiple databases at once.
+func (m *mcpDBConfigCommand) updateClientConfig(dbURI mcp.ResourceURI) error {
+	config, err := m.clientConfig.loadConfig()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	preexistentDB, commandChanged, err := m.addDatabaseToConfig(config, dbURI)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if err := config.Save(claude.FormatJSONOption(m.clientConfig.jsonFormat)); err != nil {
+		return trace.Wrap(err)
+	}
+
+	templateData := struct {
+		Name             string
+		ConfigPath       string
+		ConfigName       string
+		PreexistentDB    bool
+		CommandChanged   bool
+		OverwriteCommand bool
+	}{
+		Name:             dbURI.GetDatabaseServiceName(),
+		ConfigPath:       config.Path(),
+		ConfigName:       mcpDBConfigName,
+		PreexistentDB:    preexistentDB,
+		CommandChanged:   commandChanged,
+		OverwriteCommand: m.overwriteCommand,
+	}
+
+	return trace.Wrap(mcpDBConfigMessageTemplate.Execute(m.cf.Stdout(), templateData))
+}
+
+// addDatabaseToConfig adds the provided database, merging with existent
+// databases configured. This function returns a additional message to be
+// displayed to users.
+func (m *mcpDBConfigCommand) addDatabaseToConfig(config claudeConfig, dbURI mcp.ResourceURI) (bool, bool, error) {
+	var (
+		dbs            []string
+		updated        bool
+		commandChanged bool
+		server         = makeLocalMCPServer(m.cf, nil /* args */)
+	)
+	if existentServer, ok := config.GetMCPServers()[mcpDBConfigName]; ok {
+		// We don't place any environment variable on the MCP config, meaning if
+		// it is not empty, the user manually edited the server config.
+		// For those cases we want to keep their changes. However, in case they
+		// want a "fresh start" they can provide a flag.
+		if len(existentServer.Envs) != 0 {
+			commandChanged = true
+			if !m.overwriteCommand {
+				server.Envs = existentServer.Envs
+			}
+		}
+
+		for _, arg := range existentServer.Args {
+			// We're only interested in resources, any flags or other command
+			// parts will be discarded.
+			uri, err := mcp.ParseResourceURI(arg)
+			if err != nil {
+				continue
+			}
+
+			if !uri.IsDatabase() {
+				return false, false, trace.BadParameter("")
```

Comment:
is `trace.BadParameter("")` intentional?

Reply:
Added message.